### PR TITLE
Linear Search added and readme updated

### LINKED ISSUE #602: Linear Search
Hi I am Malavika R Vikraman , participating in GSSOC'21 .
I would like to contribute by adding Linear Search in Python to this project.

Kindly assign this issue to me!
@anubhavitis @dheerajkotwani @tarun26091999 @todi-2000 @geekquad @plazzy99

### PR DESCRIPTION
**Pull request by vedant-jain03**
Fixes #602 

--> Added Linear Search.cpp
--> Updated Readme.md

@raksha009 @tarun26091999 
Please review

### CODE REVIEW

**Review by tarun26091999 — CHANGES_REQUESTED** [review]
resolve the conflict and check space complexity

**Review by vedant-jain03 — COMMENTED** [review]
Space Complexity O(n) -> O(1)

**Review by vedant-jain03 — COMMENTED** [review]
Unchanged Readme.md file

**Review by tarun26091999 — CHANGES_REQUESTED** [review]
link it to the folder

**Comment by vedant-jain03** [comment]
> link it to the folder

@tarun26091999 
Sir actually I am not able to understand that I already have add my .cpp code in Code folder
So what else we have to link to the folder that you have requested
I am bit confuse about this part

**Comment by tarun26091999** [comment]
read the guidelines or you can see the merged PRs.

**Review by vedant-jain03 — COMMENTED** [review]
Updated Linked list.cpp and linked it to folder in Readme.md file

File: Code/C++/Linear_search.cpp [hunk 1]
```diff
@@ -0,0 +1,63 @@
...
+
+#include <iostream>
+#include <algorithm>
+#include <vector>
+
+using namespace std;
+int linear_search(int arr[],int n,int element)
+{
+    int i=0;
+    for(int i=0;i<n;i++)
```
Comment (tarun26091999) [inline Code/C++/Linear_search.cpp:17]: multiple declaration of i

**Review by tarun26091999 — CHANGES_REQUESTED** [review]

File: Algorithm/Searching_Sorting/readme.md [hunk 2]
```diff
@@ -50,6 +50,7 @@ is one of the techniques for making the elements ordered.
 
 Following are the types of searches which we will be discussing in this book.
 
+* Linear Search ----> [C++](/Code/C++/Linear_search.cpp)
```
Comment (tarun26091999) [inline Algorithm/Searching_Sorting/readme.md:53]: remove this line and link the code to unordered linear search.

**Review by vedant-jain03 — COMMENTED** [review]
Updated

**Review by tarun26091999 — APPROVED** [review]

**Comment by vedant-jain03** [comment]
@tarun26091999 Please I guess you forgot to label and also thankyou for approving

**Pull request merged**